JavaScript/JavaScriptSDK/AppInsights.ts

```diff
@@ -68,6 +68,7 @@ module Microsoft.ApplicationInsights {
                 sessionRenewalMs: () => this.config.sessionRenewalMs,
                 sessionExpirationMs: () => this.config.sessionExpirationMs,
                 endpointUrl: () => this.config.endpointUrl,
+                isInternalEndpointUrl: () => Util.isInternalApplicationInsightsEndpoint(this.config.endpointUrl),
```

Comment:
it's a different config - it's a place where customer facing IConfig is transformed into ITelemetryConfig used for internal needs. check other properties - they also rely on some internal logic like isBeaconEnabled. isInternalEndpointUrl is not exposed in IConfig.

Reply:
sorry about the late reply Yulia. Was just wondering if this needs to be exposed through ITelemetryConfig at all. Simply because its not something we need to expose for updates.

Reply:
i see... removed it from there and calling Util.isInternalApplicationInsightsEndpoint directly.

Reply:
Cool, thanks!, PR is approved already.